inspirehep/modules/deposit/dojson/fields/literature.py

```diff
@@ -221,7 +221,7 @@ def languages(self, key, value):
     if value not in ('en', 'oth'):
         return [unicode(dict(languages).get(value))]
     else:
-        return [value]
+        return []
```

Comment:
I believe the problem is really that there is the language mentioned in the title (rather than as a field)